### ISSUE TITLE
Vendored `libnfc` still require `libnfc.so.6` at runtime.

### ISSUE THREAD

**Issue by hansl** [alexrsagen/rs-nfc1-sys#9]
I'm using the latest version on crates.io (0.5.2) with default features (even if I force enable `vendored` feature it's the same issue).

I'm working on an embedded platform (DE10-Nano, essentially an ARMv7). I was looking at this crate because of the vendored feature.

The way I build: `cross build --target armv7-unknown-linux-gnueabihf --bin nfc-tester`

The output when I run this on the device itself:

```
/media/fat# ./nfc-tester
./nfc-tester: error while loading shared libraries: libnfc.so.6: cannot open shared object file: No such file or directory
```

Adding a specially compiled version of `libnfc.so.6` to `/usr/lib` fixes the issue (naturally). Is this expected?

**Comment by hansl** [comment]
Addendum: I just realized I was using the `nfc1` crate. However using the `nfc1-sys` crate directly results in the same issue.

**Comment by alexrsagen** [comment]
Please do let me know if it's working as of `v0.3.6`, @hansl 😄

**Comment by hansl** [comment]
@alexrsagen thanks for the quick fix. I have a different issue now; it says it cannot find the static library `nfc`. Is there a debian dependency I have to install? I thought it would be built from source...

```
error: could not find native static library `nfc`, perhaps an -L flag is missing?
```

**Comment by alexrsagen** [comment]
Should be fixed in `v0.3.9` @hansl, please give it a try and let me know if it works for you 😄

**Comment by hansl** [comment]
Works perfectly now. Thanks!
